CONTRIBUTING.md

```diff
@@ -41,11 +49,39 @@ top of things.
     The first line is a real life imperative statement with a ticket number
     from our issue tracker.  The body describes the behavior without the patch,
     why this is a problem, and how the patch fixes the problem when applied.
-````
+```
 
 * Make sure you have added the necessary tests for your changes.
 * Run _all_ the tests to assure nothing else was accidentally broken.
 
+### Running the tests
+
+In order to run the local test suite, you will first need to have a PostgreSQL
+instance [configured][configure_postgres], and will need to create the test
+users:
+
+    $ createuser -DRSP pdb_test
+    $ createuser -dRsP pdb_test_admin
+
+You will also need to set the following environment variables if the
+default values aren't appropriate:
+
+  * PDB\_TEST\_DB\_HOST (defaults to localhost)
```

Comment:
So this was already a problem but since you're fixing this anyways, could you remove these backslashes and code block these variables `PDB_TEST_DB_HOST` for example.